build.cake

```diff
@@ -63,35 +86,210 @@ Task("Test")
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netstandard2.0",
+					Runtime = "win-x86",
+					OutputDirectory = build.ToString() + "/runtimes/win-x86/lib/netstandard2.0",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netstandard2.0",
+					Runtime = "win-x64",
+					OutputDirectory = build.ToString() + "/runtimes/win-x64/lib/netstandard2.0",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+
+			// .NET Core 2.1
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netcoreapp2.1",
+					OutputDirectory = build.ToString() + "/lib/netcoreapp2.1",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netcoreapp2.1",
+					Runtime = "win-x86",
+					OutputDirectory = build.ToString() + "/runtimes/win-x86/lib/netcoreapp2.1",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netcoreapp2.1",
+					Runtime = "win-x64",
+					OutputDirectory = build.ToString() + "/runtimes/win-x64/lib/netcoreapp2.1",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+			}
     });
 
 Task("Package-NuGet")
-    .IsDependentOn("Update-Version")
+    .IsDependentOn("Publish")
     .Description("Generates NuGet packages for each project")
     .Does(() =>
     {
-        foreach(var project in GetFiles("./src/Serilog.Sinks.Oracle/Serilog.Sinks.Oracle/*.csproj"))
-        {
-            Information("Packaging " + project.GetFilename().FullPath);
+		 
+		 if ( !DirectoryExists(nuget) )
+		 {
+			CreateDirectory(nuget);
+		 }
 
-            var content =
-                System.IO.File.ReadAllText(project.FullPath, Encoding.UTF8);
+	     var nuGetPackSettings   = new NuGetPackSettings {
+                                     Id                      = "Serilog.Sinks.Oracle",
+                                     Version                 =  version,
+                                     RequireLicenseAcceptance= false,
+                                     Symbols                 = false,
+                                     NoPackageAnalysis       = true,
+                                     Files                   = new [] {
+                                            new NuSpecContent {Source = "**", Target = "."},
+																	   },	
+									Dependencies			 = new [] {
+											// .NETFramework4.5.2
```

Comment:
I try to not surpass a max of 120 columns because there is people who have small screens/small resolutions and force then to scroll left/right to read all the code (well, considering they let the tests window opened, solution opened, etc).

If you want to have a discussion about this style point, would be great to have it an feel free to open a sepated issue for it, probably would be easier for others to interact over there.

Reply:
Fixed